tray/src/com/dvt/main/creational/abstract_factory/doors/Model1Doors.java

```diff
@@ -0,0 +1,16 @@
+<<<<<<< HEAD
+package com.dvt.creational.abstract_factory.doors;
+
+import com.dvt.creational.abstract_factory.Part;
+=======
+package creational.abstract_factory.doors;
+
+import creational.abstract_factory.Part;
+>>>>>>> main
```

Comment:
Please fix merge conflict.
```suggestion
<<<<<<< HEAD
package com.dvt.creational.abstract_factory.doors;

import com.dvt.creational.abstract_factory.Part;
=======
package creational.abstract_factory.doors;

import creational.abstract_factory.Part;
>>>>>>> main
```